samples/openapi3/client/petstore/dart-dio-next/petstore_client_lib_fake/lib/src/model/additional_properties_class.dart

```diff
@@ -46,13 +46,13 @@ class _$AdditionalPropertiesClassSerializer implements StructuredSerializer<Addi
             result
                 ..add(r'map_property')
                 ..add(serializers.serialize(object.mapProperty,
-                    specifiedType: const FullType(BuiltMap, [FullType(String), FullType(String)])));
+                    specifiedType: const FullType(BuiltMap, [const FullType(String), const FullType(String)]) ,),);                
```

Comment:
Can you try to remove the additional `const` keywords which may generate analyzer warnings for some users?
And removing the additional whitespaces and commas would also be great for the diff.

Reply:
for the commas, it's good if users (like me) are formatting the generated output

Reply:
I agree but can we do that in a separate PR?

Reply:
sure, why not